Add useful plugins

bboulesteix opened the pull request:
This pull request:  closes #9, closes #10, and closes #11.

bboulesteix committed:
Add a few gems to the gemfile

This commit adds 4 new gems to the gemfile.

bboulesteix committed:
Add favicons

This commit adds favicon file at the root of the repo.

bboulesteix committed:
Add /favicon folder

This commit adds more favicon file to the repo.

bboulesteix committed:
Add default social card image

This commit adds a default social card image that will be used as a preview for all our pages (for the time being).

bboulesteix committed:
Increase size of external navigation link

This commit increases the size of the link redirecting to Memo Bank's marketing website from the header of our growth guide.

bboulesteix committed:
Increase line height

This commit removes custom SCSS in order for our style to fallback on the theme's default style, which comes with great line height by default.

bboulesteix committed:
Update _config.yml

This commit changes the default language of our website, sets our SEO plugin up, and adds the name of our gem to the config file (because GitHub Pages ignores them otherwise).

dirtyhenry commented on _config.yml:
```diff
@@ -39,15 +66,24 @@ kramdown:
   auto_ids: true
   syntax_highlighter: rouge
 
+# Permalin
```
Typo, missing a k.

bboulesteix replied:
Oh thanks for pointing that typo out. 👍

dirtyhenry commented on Gemfile:
```diff
@@ -34,4 +38,5 @@ gem "wdm", "~> 0.1.1", :platforms => [:mingw, :x64_mingw, :mswin]
 
-gem "webrick", "~> 1.7"
+# Add webrick to fix a bug with Ruby version 3.0.0 or higher
+gem "webrick", "~> 1.7"
```
Could you add a blank line at the end of this file? 🙏

bboulesteix replied:
Yes, will do. Thanks @dirtyhenry. 👍

bboulesteix committed:
Fix typo in comment

This commit fixes a typo in a comment and adds a blank line at the end of the gemfile.
